docs/oscar_references.bib

```diff
@@ -378,6 +401,33 @@ @Article{BS09
   doi           = {10.1016/j.aim.2009.06.009}
 }
 
+@Article{BS22,
+  author        = {Brandhorst, Simon and Shimada, Ichiro},
+  title         = {Automorphism groups of certain {E}nriques surfaces},
```

Comment:
```suggestion
  title         = {Automorphism groups of certain Enriques surfaces},
```

Reply:
At least in my personal bibtex files I keep this as is, to keep capitalisation?

Reply:
If using with biber or biblatex (e.g. in the .bib file of a paper), you need the braces to preserve casing. If using with DocumenterCitations.jl (e.g. here), please remove them, since it preserves casing anyway, but sometimes braces end up in the output.

Reply:
good to know, thx